webapp/components/CommentList/CommentList.vue

```diff
@@ -12,9 +12,11 @@
         :comment="comment"
         :post="post"
         :routeHash="routeHash"
+        class="comment-tag"
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-01-22T15:12:31Z" title="Wednesday, January 22nd 2020, 4:12:31 pm +01:00">Jan 22, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -7,6 +7,7 @@
     <ds-space margin-bottom="large" />
     <div v-if="post.comments && post.comments.length" id="comments" class="comments">
       <comment
+        @reply="reply"
```


Move to end of list …